LogoREGICore/Sources/Infra/SwiftData/OrderSwiftData.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  OrderSwiftData.swift
+//  cafelogos-pos
+//
+//  Created by Devin AI on 2025/03/06.
+//
+
+import Foundation
+import SwiftData
+
+@MainActor
+public final class OrderSwiftData: OrderRepository {
+    private let modelContainer: ModelContainer
+    
+    public init(modelContainer: ModelContainer) {
+        self.modelContainer = modelContainer
+    }
+    
+    public func save(order: Order) async {
+        let context = modelContainer.mainContext
+        
+        do {
+            let descriptor = FetchDescriptor<OrderModel>(
+                predicate: #Predicate {
+                    $0.id == order.id
+                }
+            )
```

Comment:
まだこの箇所でエラーが出るので、次のように修正してみてはどうでしょうか？
同じ修正を他の同じ`save`箇所でも行ってください。
```suggestion
        let context = modelContainer.mainContext
        let paymentId = payment.id
        
        do {
            let descriptor = FetchDescriptor<PaymentModel>(
                predicate: #Predicate {
                    $0.id == paymentId
                }
            )
```